config/constants.ts

```diff
@@ -99,6 +99,24 @@ Resources required for BaseSpace TES stack
 
 export const basespaceAccessTokenSecretName = '/manual/BaseSpaceAccessTokenSecret'; // pragma: allowlist secret
 
+/*
+ICAv1 Resources
+ */
+export const icav1AccessTokenSecretName: Record<AppStage, string> = {
+  [AppStage.BETA]: 'ICAv1JWTKey-umccr-prod-service-trial', // pragma: allowlist secret
```

Comment:
I don't think these keys exist,

Reply:
No, they don't, draft still, working on it.